Diplomatic.Core/Classes/PDFGenerator.cs

```diff
@@ -28,11 +29,11 @@ public class PDFGenerator : IDiplomaGenerator
         public IDiploma Generate(Template template)
         {
             PdfDocument document = new PdfDocument();
-            document.Info.Title = "Your diploma is ready";
-            document.Info.Author = "Diplomatic 2018";
+            document.Info.Title = template.TemplateName;
+            document.Info.Author = "Diplomatic © 2018";
             document.PageLayout = PdfPageLayout.SinglePage;
 
-            XPdfForm originalPdf = XPdfForm.FromFile(template.FilePath);
+            XPdfForm originalPdf = XPdfForm.FromStream(template.RawTemplate.Stream);
```

Comment:
Isn't this conflicting with the Law of Demeter? Perhaps create a property `Stream` on the `Template`-class and hide the `RawTemplate`-attribute.